### ISSUE TITLE
#1588	de kleur van tekstlagen ook aanpassen per laag

### ISSUE THREAD

**Issue by ElisabethKloren** [bimloket/NLCS#25]
#1588	Het is wenselijk dat de kleur van tekstlagen ook aan te passen zijn per laag, net zoals dat dit voor geometrie lagen mogelijk is. Teksten van bestaande hoogtes drukken wij grijs af en nieuwe hoogtes in het zwart. Het lijkt ook een tegenstrijdigheid in de NLCS beschrijving aangezien tekstlagen wel een status krijgen, maar die kunnen vervolgens per status niet in kleur worden aangepast.

**Comment by ElisabethKloren** [comment]
Reactie Expertcommissie
"volgens tabel zijn de kleuren per letterhoogte vastgelegd, echter wordt onder 3.1 ook aangegeven
e. Kleuren, lijnstijlen en –dikten moeten altijd in de lagenstructuur worden geregeld. Objecten hebben in 
beginsel geen eigen kleur, lijnstijl of lijndikte; slechts bepaalde symbolen hebben een eigen kleur, 
lijnstijl en/of –dikte. "

**Comment by SanderNijhof** [comment]
Ja het zou duidelijker zijn als bij paragraaf 3.5.9 bij alle objecten in de kolom 'KLEUR' 'objecttabellen' zou staan.

**Comment by MvanderHulst** [comment]
Als gebruiker zou ik graag op laagniveau de kleur van een tekst-laag willen kunnen regelen. De regel dat een kleur aan een tekstgrootte vastzit, is hierbij de beperkende factor. Ondertussen ben ik hier wel aan gewend, maar veel van mijn 'klanten' klagen hier ook over.
Op basis van de melding, is te herleiden dat de gebruiker geen 'layer override' op XREF's toepast, maar de bron wilt aanpassen. Hierbij geeft de NLCS ook aan dat de presentatie altijd grijs gemaakt kan worden met een override en dit niet handig is om in de bron te doen. Het probleem is dus een combinatie van een beperking in de standaard en de werkwijze van de melder.

**Comment by MvanderHulst** [comment]
Klopt, maar alle lagen voor een tekst met teksthoogte 18 worden standaard met kleur 10 aangemaakt.
Als ik daarna met de hand de kleur van de layer aanpas naar bijvoorbeeld 'blauw'. Is dit een afwijking om de standaard, omdat de layer van de tekstlaag geen kleur 10 meer heeft.
Dus ja... het wordt op layer niveau geregeld, maar de instelling staat wel voorgeschreven in de standaard.

Diverse gebruikers zouden hier vrijheid in willen, dat de relatie tussen tekstgrootte en kleur er niet meer is.
Zelf ben ik hierin Zwitserland, en heb er geen specifieke mening over.

**Comment by ElisabethKloren** [comment]
Sander zegt: Alleen wijziging Formele beschrijving. Kleur wordt (altijd) geregeld in de laageigenschappen, tenzij... (voorb. Verkeersborden).

**Comment by ElisabethKloren** [comment]
EC 2023-11-09 
Probleem: in model wil je met kleuren verschillen zien tussen teksten, in plot wil je teksten in zwart afdrukken

Is idee om de eerste 9 kleuren van autocad / die nu niet worden gebruikt in de standaard (check; niet 7 die gebruiken we wel) te gebruiken voor teksten en annotaties? Dan worden die teksten zwart afgedrukt maar hebben wel kleur in het model

NLCS colortabel plotstyle moet aangepast worden, de eerste 9 alleen in kleur zwart (plotten in zwart)

Checken wat we wel gebruiken, annotaties: teksten, aanpijlingen, en maatvoering (is al 7) 1t/m9 codes geven
